lib/caruby2go.rb

```diff
@@ -29,6 +29,10 @@ def parkingspots # placemarks
   def vehicles # placemarks
     issue_get(build_uri('vehicles'))
   end
+  
+  def operationareas # placemarks
+    issue_get(build_uri('operationareas'))
```

Comment:
I need to adjust houndci settings -- I prefer single quotes until I need to interpolate